public/index.html

```diff
@@ -115,6 +132,50 @@ <h2>PayJoin here to open these channels</h2>
 				channels.removeChild(channels.lastElementChild);
 			}
 
+			async function get_recommended_channels(){
+				document.getElementsByClassName('recommendation-button-wrapper')[0].classList.add('invisible')
+				try {
+					recommendations = await (await fetch('/recommend', {
+						method: 'GET',
+					})).json()
+
+					for (var node_type in recommendations) {
+						let text = `
+						<div>
+							<h3>${node_type}</h3>
+							<small>a well connected node that is reliable for payments</small>
+							<br>
+							<br>
+							<ul>
+								<li>${recommendations[node_type].alias}</li>
+								<li>&nbsp;Active Capacity:&nbsp;₿${recommendations[node_type].capacity / SATS_PER_BTC} | &nbsp;Channel Count:&nbsp;${recommendations[node_type].active_channel_count}</li> 
+							</ul>
+							<div>
+								<button class="colored-button" type="button" onclick="add_recommended_node('${node_type}')">Use</button>	
+							</div>
+						</div>
+						<br>
+						`;
+						document.getElementById("recommendations").insertAdjacentHTML("beforeend", text);
+					}
+				} catch (error) {
+					console.error(error)
+					alert("Sorry, can't connect to node sources right now.")
+				}
+			}
+
+			function add_recommended_node(type = 'routing_node') {
+				const node = recommendations[type];
```

Comment:
I'm not sure we should create and reference non-view global state

Reply:
We want to cache the API payload somewhere... any suggestions? This seems like the most naive approach